feature/space/src/main/java/com/yapp/itemfinder/space/selectspace/SelectSpaceState.kt

```diff
@@ -0,0 +1,17 @@
+package com.yapp.itemfinder.space.selectspace
+
+import com.yapp.itemfinder.domain.model.Data
+import com.yapp.itemfinder.feature.common.State
+
+sealed class SelectSpaceState : State {
+    object Uninitialized : SelectSpaceState()
+    object Loading : SelectSpaceState()
+    data class Success(
+        val dataList: List<Data>,
```

Comment:
프로퍼티 이용하여 fetch 필요한 경우에만 하도록 수정
```suggestion
        val isRefreshNeed: Boolean,
```